src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataEntityReferenceLinkDeserializer.cs

```diff
@@ -3,16 +3,22 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
 using System.Threading.Tasks;
 using Microsoft.AspNet.OData.Common;
+using Microsoft.AspNet.OData.Interfaces;
+using Microsoft.AspNet.OData.Routing;
 using Microsoft.OData;
+using Microsoft.OData.Edm;
 
 namespace Microsoft.AspNet.OData.Formatter.Deserialization
 {
     /// <summary>
     /// Represents an <see cref="ODataDeserializer"/> that can read OData entity reference link payloads.
```

Comment:
Maybe update the doc comment to match the argument type

Reply:
The doc is okay..The ODataEntityReferenceLinkDeserializer is a representation of the ODataDeserializer.